docs/rules/sort-comp.md

```diff
@@ -24,6 +24,15 @@ var Hello = createReactClass({
 });
 ```
 
+```jsx
+class Hello extends React.Component {
+  render () {
```

Comment:
```suggestion
  render() {
```
whoops, sorry, i think i spaced it wrong in my original suggestion. same throughout.